bin/seed-csv.js

```diff
@@ -0,0 +1,29 @@
+import commander from 'commander';
+import fs from 'fs';
+import csvm from '../app/models/CSVModels.js';
+import {rows} from '../app/models/CSVModels.js';
+
+const { program } = commander;
+
+program
+// cruft templating...
+//  .option('-p, --path <string>', 'path to assets')
+//  .option('-n, --name <string>', 'deck name')
+//  .option('-d, --description <string>', 'deck description')
+  .parse(process.argv);
+
+(async () => {
```

Comment:
You wouldn't typically combine `async/await` with `then/catch`. You can instead `try/catch` any `await` calls inside the `async` function.